test/regression/RegressionTest.cc

```diff
@@ -33,7 +33,7 @@ void RegressionTest::run(const char* source, const char* triple,
   Config::set(config);
 
   const size_t memorySize =
-      Config::get()["Simulation-Memory"]["Size"].as<size_t>();
+      Config::get()["Memory-Hierarchy"]["L1-Data"]["Size"].as<size_t>();
```

Comment:
Add a comment why size of memory is L1-Size and not DRAM-size

Reply:
Changed config to use DRAM rather than L1 to avoid confusion